src/Symfony/Component/Serializer/Mapping/Loader/AttributeLoader.php

```diff
@@ -88,33 +77,35 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                 $classMetadata->addAttributeMetadata($attributesMetadata[$property->name]);
             }
 
+            $attributeMetadata = $attributesMetadata[$property->name];
             if ($property->getDeclaringClass()->name === $className) {
                 if ($classContextAnnotation) {
-                    $this->setAttributeContextsForGroups($classContextAnnotation, $attributesMetadata[$property->name]);
+                    $this->setAttributeContextsForGroups($classContextAnnotation, $attributeMetadata);
                 }
 
                 foreach ($classGroups as $group) {
-                    $attributesMetadata[$property->name]->addGroup($group);
+                    $attributeMetadata->addGroup($group);
                 }
 
                 foreach ($this->loadAttributes($property) as $annotation) {
+                    $loaded = true;
```

Comment:
Is this variable still relevant? It doesn't seem used anywhere in its scope

Reply:
Yes, it is returned at the end of the method.

Reply:
Oh right, my bad